doc/source/whatsnew/v2.1.0.rst

```diff
@@ -643,6 +643,8 @@ Other
 - Bug in :meth:`DataFrame.reindex` with a ``fill_value`` that should be inferred with a :class:`ExtensionDtype` incorrectly inferring ``object`` dtype (:issue:`52586`)
 - Bug in :meth:`DataFrame.shift` and :meth:`Series.shift` and :meth:`DataFrameGroupBy.shift` when passing both "freq" and "fill_value" silently ignoring "fill_value" instead of raising ``ValueError`` (:issue:`53832`)
 - Bug in :meth:`DataFrame.shift` with ``axis=1`` on a :class:`DataFrame` with a single :class:`ExtensionDtype` column giving incorrect results (:issue:`53832`)
+- Bug in :meth:`GroupBy.first` the helper function should return the NA value of the EA if all values are NA (:issue:`39098`)
+- Bug in :meth:`GroupBy.last` the helper function should return the NA value of the EA if all values are NA (:issue:`39098`)
```

Comment:
```suggestion
- Bug in :meth:`GroupBy.first` and :meth:`GroupBy.last` where an empty group would return ``np.nan`` instead of a an ExtensionArray's NA value (:issue:`39098`)
```

Reply:
Will do.